libraries/chain/include/steemit/chain/comment_object.hpp

```diff
@@ -168,6 +169,8 @@ namespace steemit { namespace chain {
          bool              allow_replies = true;      /// allows a post to disable replies.
          bool              allow_votes   = true;      /// allows a post to receive votes;
          bool              allow_curation_rewards = true;
+
+         bip::vector< std::pair< account_name_type, uint16_t >, allocator< std::pair< account_name_type, uint16_t > > > beneficiaries;
```

Comment:
use shared_vector<beneficiary_type>  and define beneficiary type as { account_name, percent_type }